policy-engine/src/config/file.rs

```diff
@@ -18,6 +18,9 @@ pub struct FileOptions {
     /// Upstream options.
     pub upstream: Option<UpstreamOptions>,
 
+    /// Policy plugins options.
+    pub policy: Option<Vec<toml::Value>>,
```

Comment:
@steveeJ I was unsure whether to carve out `Vec<toml::Value>` to its own type (assuming serde allows me to make it transparent again when deserializing).

Reply:
I think it doesn't warrant for a new type.